ampligraph/latent_features/models/EmbeddingModel.py

```diff
@@ -1022,22 +1025,15 @@ def configure_evaluation_protocol(self, config=None):
 
             - **corruption_entities**: List of entities to be used for corruptions.
               If ``all``, it uses all entities (default: ``all``)
-            - **corrupt_side**: Specifies which side to corrupt. ``s``, ``o``, ``s+o`` (default)
-            - **default_protocol**: Boolean flag to indicate whether to use default protocol for evaluation.
-              This computes scores for corruptions of subjects and objects and ranks them separately.
-              This could have been done by evaluating s and o separately and then
-              ranking but it slows down the performance.
-              Hence this mode is used where s+o corruptions are generated at once but ranked separately for speed up
+            - **corrupt_side**: Specifies which side to corrupt. ``s``, ``o``, ``s+o``, ``s,o`` (default)
+              In 's,o' mode, s+o corruptions are generated at once but ranked separately for speed up
```

Comment:
Unclear, can you change to: 

"In 's,o' mode subject and object corruptions are generated .. "